CommandKeyList.md

```diff
@@ -1,61 +1,63 @@
-
-| Key | Default Mapping |
-| ---: | :--- |
-| - | Decrement (Skill/Spell Points, Advantage Level, Equipment Qty) |
-| = | Increment (Skill/Spell Points, Advantage Level, Equipment Qty) |
-| ' | Rotate Equipment State |
-| , | Preferences |
-| [ | Decrement Tech Level |
-| ] | Increment Tech Level |
-| A | Select All |
-| B | New Spell |
-| C | Copy |
-| D | New Advantage |
-| E | New Equipment |
-| F | |
-| G | Open Page Reference |
-| H | Hide Application (Mac) |
-| I | Open Detail Editor |
-| J | Jump to Search |
-| K | New Skill |
-| L | Export to GURPS Calculator |
-| M | New Note |
-| N | New Character Sheet |
-| O | Open |
-| P | Print |
-| Q | Quit |
-| R | |
-| S | Save |
-| T | New Technique |
-| U | Duplicate |
-| V | Paste |
-| W | Close |
-| X | Cut |
-| Y | Redo |
-| Z | Undo |
-| SHIFT+A | Apply Template to Character Sheet |
-| SHIFT+B | New Spell Container |
-| SHIFT+C | Copy to Character Sheet |
-| SHIFT+D | New Advantage Container |
-| SHIFT+E | New Equipment Container |
-| SHIFT+F | |
-| SHIFT+G | Open Each Page Reference |
-| SHIFT+H | |
-| SHIFT+I | Generate Random Female Name |
-| SHIFT+J | |
-| SHIFT+K | New Skill Container |
-| SHIFT+L | |
-| SHIFT+M | Note Note Container |
-| SHIFT+N | |
-| SHIFT+O | |
-| SHIFT+P | Page Setup |
-| SHIFT+Q | |
-| SHIFT+R | |
-| SHIFT+S | Save As |
-| SHIFT+T | Copy to Template |
-| SHIFT+U | |
-| SHIFT+V | Generate Random Male Name |
-| SHIFT+W | |
-| SHIFT+X | |
-| SHIFT+Y | |
-| SHIFT+Z | |
+
+| Key | Default Mapping |
+| ---: | :--- |
+| - | Decrement (Skill/Spell Points, Advantage Level, Equipment Qty) |
+| = | Increment (Skill/Spell Points, Advantage Level, Equipment Qty) |
+| CTRL+SHIFT+= | Skill Level Increment |

```

Comment:
The CTRL or CMD key is implied in this list, as it is platform-specific. Please omit it.